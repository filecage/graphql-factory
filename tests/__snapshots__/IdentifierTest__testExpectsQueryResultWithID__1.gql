type Query {
  "Loads cars and their identifiers for reduced object type testing"
  GetCarWithScalarId(
    "The car's ID"
    id: ID!
  ): CarWithScalarId
}

type CarWithScalarId {
  id: ID!
  name: String!
}
