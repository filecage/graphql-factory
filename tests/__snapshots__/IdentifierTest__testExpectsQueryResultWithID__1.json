{
    "data": {
        "scirocco": {
            "id": "1",
            "name": "Volkswagen Scirocco"
        },
        "c43": {
            "id": "2",
            "name": "Mercedes C43"
        }
    }
}
